features/step_definitions/fincap_pages_steps.rb

```diff
@@ -0,0 +1,67 @@
+Given(/^I have an insight page layout$/) do
+  english_site = Comfy::Cms::Site.find_or_create_by(
+    label: 'en',
+    identifier: 'money-advice-service-en',
+    hostname: 'localhost:3000',
+    path: 'en',
+    locale: 'en',
+    is_mirrored: true
+  )
+
+  english_site.layouts.find_or_create_by(
```

Comment:
The same thing with the layout, we already have a layout factory. We can add traits for this later on to DRY our implementation as the code grows to accommodate Evaluation and Review layout types.
Again it's fine to have the basics here as you've done.
It will change in the near future though.